pkg/rbac/cluster_role_binding.go

```diff
@@ -80,3 +80,11 @@ func DeleteClusterRoleBinding(c base.K8sClientWithLogging, ctx context.Context,
 
 	return nil
 }
+
+// Validate ensures the cluster role binding is not to cluster-admin
+func (crb *ClusterRoleBinding) Validate() error {
```

Comment:
This way to validate the bindings looks OK to me, but it seems that it's not used anywhere yet.